### ISSUE TITLE
[captainkube] Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:phippyandfriends:default" cannot list pods at the cluster scope

### ISSUE THREAD

**Issue by mathieu-benoit** [Azure/phippyandfriends#9]
I'm deploying my cluster like this:
```
$az aks create -l $location -n $aksname -g $rgname --generate-ssh-keys -k 1.11.3 -s Standard_DS2_v2 -c 1 --enable-addons monitoring,http_application_routing
```
Then deploying `parrot` and `captainkube` like this:
```
$helm upgrade --namespace phippyandfriends --install --force --set ingress.basedomain=xxx.eastus.aksapp.io --set image.repository=mabenoit/parrot --set image.tag=522 --wait parrot parrot-522.tgz
$helm upgrade --namespace phippyandfriends --install --force --set image.repository=mabenoit/captainkube --set image.tag=523 --wait captainkube captainkube-523.tgz
```
The deployment looks good:
```
$kubectl get all -n phippyandfriends
NAME                                           READY     STATUS    RESTARTS   AGE
pod/captainkube-captainkube-869b6fdf77-gcvh6   1/1       Running   0          17s
pod/parrot-parrot-5d85fcf9b9-lzczx             1/1       Running   0          6h

NAME                    TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)   AGE
service/parrot-parrot   ClusterIP   x.x.x.x   <none>        80/TCP    6h

NAME                                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/captainkube-captainkube   1         1         1            1           6h
deployment.apps/parrot-parrot             1         1         1            1           6h

NAME                                                 DESIRED   CURRENT   READY     AGE
replicaset.apps/captainkube-captainkube-869b6fdf77   1         1         1         6h
replicaset.apps/parrot-parrot-5d85fcf9b9             1         1         1         6h
```
But the app is not working. I got this error message by doing `kubectl logs captainkube-podname -n phippyandfriends`:
```
main.go:124: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:phippyandfriends:default" cannot list pods at the cluster scope
```

**Comment by mathieu-benoit** [comment]
Other than not using `draft up` by using `helm` directly, I'm investigating if I'm missing something or if there is an issue.
FYI @sabbour and @bradygmsft

**Comment by mathieu-benoit** [comment]
To resolve this `forbidden` error, I had to run:
```
kubectl create clusterrolebinding default-view --clusterrole=view --serviceaccount=phippyandfriends:default
```
I found this documented [here](https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration#authenticating-inside-the-cluster).

**Comment by sabbour** [comment]
Thanks! We need to update the docs for RBAC enabled clusters.

**Comment by sabbour** [comment]
@mathieu-benoit can you do a PR to update the docs with your resolution? :)

**Comment by mathieu-benoit** [comment]
@sabbour, yep will do!

**Comment by mathieu-benoit** [comment]
Taken into account and merged into master via PR #11
